src/sentry/integrations/bitbucket/repository.py

```diff
@@ -67,7 +67,7 @@ def create_repository(self, organization, data):
                         '/extensions/bitbucket/organizations/{}/webhook/'.format(organization.id)
                     ),
                     'active': True,
-                    'events': ['repo:push'],
+                    'events': ['repo:push', 'pullrequest:fulfilled'],
```

Comment:
is this going to be problematic for the existing bitbucket user(s)? we should figure out what to tell them to do to ensure this feature will work for them going forward

cc @saragilford

Reply:
@macqueen yes. For anyone who had enabled repos for their integration before this change, thy will not have the pullrequest webhook. We could possibly create a script to add their missing webhook manually? Good catch.

Reply:
This will be the same for vsts change as well.